infra/terraform/file_transfer.go

```diff
@@ -60,8 +60,10 @@ func (t *terraform) makeRemoteCommand(fileUrl, command string) (string, error) {
 		echo Downloading installer %[5]s to %[3]s ... && %[2]s && \
 		echo Creating installer dir && mkdir -p %[1]s/installer && \
 		echo Unpacking installer && tar -xvf %[3]s -C %[1]s/installer && \
+		echo Checking existence of post-downloading installer script and executing it && \
```

Comment:
Can you change this to a `text/tempate` - I'm not comfortable with more than 3 unnamed arguments.

Reply:
Ok, will do